setup.py

```diff
@@ -53,7 +53,7 @@
     'invenio-assets>=1.0.0b2',
     'invenio-base>=1.0.0a11',
     'invenio-celery>=1.0.0a4',
-    'invenio-classifier~=1.1.0',
+    'invenio-classifier~=1.0,>=1.1.0',
```

Comment:
yep, if the project does not follow semantic versioning it's useless.
So this should be pinned to <1.2?

I found that docutils should be pinned to <0.14 too, or with our current sphinx version, it fails like https://github.com/coala/coala/issues/4280